CHANGES.rst

```diff
@@ -1,40 +1,57 @@
-Changelog
-=========
```

Comment:
Why does this file not get its own title?  We should drop any other title in another location.

Reply:
Fixed in 18257b2247f82487b187e368b737e07be9ba1289